projects/packages/search/src/classic-search/class-classic-search.php

```diff
@@ -101,7 +101,10 @@ protected function __construct() {
 	 * @return static The class singleton.
 	 */
 	public static function instance( $blog_id = null ) {
-		if ( null !== $blog_id && ! isset( self::$instance ) ) {
+		if ( ! isset( self::$instance ) ) {
+			if ( null === $blog_id ) {
+				$blog_id = get_current_blog_id();
```

Comment:
sorry to be a pain. Just a bit confused by how we get `blog_id`. AFAIK, `get_current_blog_id` right? If we need this again:
```
defined( 'IS_WPCOM' ) && constant( 'IS_WPCOM' ) ) ? get_current_blog_id() : Jetpack_Options::get_option( 'id' )
```

Could we extract it into `Helper` class or somewhere?

Reply:
Fixed!